### ISSUE TITLE
Warp 3.1.1 fails with "Not in scope: toBS"

### ISSUE THREAD

**Issue by k-bx** [yesodweb/wai#419]
Getting this build error when installing latest warp via cabal:

```
Network/Wai/Handler/Warp/IO.hs:50:17: Not in scope: ‘toBS’
```

**Comment by k-bx** [comment]
It seems to be introduced in 3.1.0

**Comment by creichert** [comment]
Thanks for the report, it looks like this was hiding in an #if statement and not picked up by the CI. Hopefully we can get this ironed out and also get coverage on all the possibilities.

**Comment by creichert** [comment]
@k-bx I just merge a fix for toBS, can you verify this is fixed for you? There might be another patch upcoming as mentioned in #420

**Comment by k-bx** [comment]
Sorry, not until next Tuesday.

**Comment by kazu-yamamoto** [comment]
Please note that Warp now does not support blaze-builder < 0.4. See #422.

**Comment by k-bx** [comment]
Works now.

**Comment by kazu-yamamoto** [comment]
Thank you for your report.
